cmd/multus-daemon/main.go

```diff
@@ -108,8 +107,9 @@ func main() {
 		os.Exit(3)
 	}
 
-	if *multusConfigFile == defaultMultusConfigFile {
-		if *cniVersion == defaultMultusCNIVersion {
+	// Generate multus CNI config from current CNI config
+	if *multusConfigFile == "auto" {
+		if *cniVersion == "" {
```

Comment:
+1. I think it's OK to reduce down to the string value in these cases because of legibility.

If there's a higher reliance on repeating these strings we could fall back to the previous method.